code/controllers/subsystems/discord.dm

```diff
@@ -317,6 +311,103 @@ SUBSYSTEM_DEF(discord)
 		send_to_admins("Server started as invisible!")
 
 
+
+/datum/controller/subsystem/discord/proc/post_webhook_event(var/tag, var/list/data)
+	set background = 1
```

Comment:
```suggestion
	set background = TRUE
```
This doesn't even look like it can sleep though, does it error out with SHOULD_NOT_SLEEP(TRUE)?